account_banking_sepa_direct_debit/data/account_payment_method.xml

```diff
@@ -5,7 +5,7 @@
         <field name="code">sepa_direct_debit</field>
         <field name="payment_type">inbound</field>
         <field name="bank_account_required" eval="False" />
-        <field name="mandate_required" eval="False" />
+        <field name="mandate_required" eval="True" />
```

Comment:
Please don't change this again.

Reply:
If mandate_required=False, it doesn't work any more !!! So I change it back... to allow users to use the module!
On the payment order line, you the mandate field is invisible when mandate_required is False, cf https://github.com/akretion/banking/blob/15-mig-account_banking_mandate/account_banking_mandate/views/account_payment_line.xml#L20
I guess it's also broken on v14...

Reply:
Then remove such invisible part, which is the nonsense, as you one thing is to be required, and the other to be hidden. I can fix it in v14 that way.